test/jdk/javax/net/ssl/templates/TLSBase.java

```diff
@@ -148,14 +147,10 @@ static class Server extends TLSBase {
         // Clients sockets are kept in a hash table with the port as the key.
         ConcurrentHashMap<Integer, SSLSocket> clientMap =
                 new ConcurrentHashMap<>();
-        Thread t;
         List<Exception> exceptionList = new ArrayList<>();
         ExecutorService threadPool = Executors.newFixedThreadPool(1,
-            r -> {
-                Thread t = Executors.defaultThreadFactory().newThread(r);
-                return t;
-            });
-
+            r -> Executors.defaultThreadFactory().newThread(r));
```

Comment:
Do you think we may simplify this to `ExecutorService threadPool = Executors.newFixedThreadPool(1);` while this line is being changed?

Reply:
Hmm.  I'll try removing it to see if I get any failures.  I don't remember why I added the threadfactory entry.